source/features/open-all-conversations.tsx

```diff
@@ -13,15 +14,12 @@ function getUrlFromItem(issue: Element): string {
 		.href;
 }
 
-// eslint-disable-next-line import/prefer-default-export
-export function confirmOpen(count: number): boolean {
```

Comment:
I'd include actually opening the tabs as part of this utility, like `openTabs(urls: string[])`

In `open-all-notifications` we can do the other DOM manipulation _after_ we open the tabs

Reply:
Not necessary as part of this PR though, you can merge it at will